.github/workflows/test.yml

```diff
@@ -147,6 +156,8 @@ jobs:
       - uses: ./ # Uses an action in the root directory
         with:
           required-ros-distributions: ${{ matrix.ros_distribution }}
+          # TODO remove this once Humble has been officially released
+          use-ros2-testing: ${{ matrix.ros_distribution == 'humble' && 'true' || '' }}
```

Comment:
Can't this just be

```
use-ros2-testing: ${{ matrix.ros_distribution == 'humble'}}
```

If we're only enabling the testing repo for humble? I don't quite understand the ``&& 'true' || ''`` part.

Reply:
It works, thanks for the suggestion!